callback.php

```diff
@@ -41,6 +41,10 @@
           
             $jwt = new XeroAPI\XeroPHP\JWTClaims();
             $jwt->setTokenId($accessToken->getValues()["id_token"]);
+
+            echo $accessToken->getValues()["id_token"];
+
+            //die();
```

Comment:
Ditto - is this troubleshooting? https://www.w3schools.com/php/func_misc_die.asp